app-vim/jedi/jedi-0.8_p20171015.ebuild

```diff
@@ -12,22 +12,28 @@ COMMIT="77924398bd594e238766153cec97ace62650f082"
 
 DESCRIPTION="vim plugin: binding to the autocompletion library jedi"
 HOMEPAGE="https://github.com/davidhalter/jedi-vim"
-SRC_URI="https://github.com/davidhalter/jedi-vim/archive/${COMMIT}.zip -> ${P}.zip"
+SRC_URI="https://github.com/davidhalter/jedi-vim/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
 
 LICENSE="MIT"
 KEYWORDS="~amd64 ~x86"
+IUSE="test"
 
 RDEPEND="dev-python/jedi[${PYTHON_USEDEP}]"
 DEPEND="app-editors/vim[python]"
 
 S="${WORKDIR}/jedi-vim-${COMMIT}"
 
-# Tests are broken.
-RESTRICT="test"
-
-# Makefile tries hard to call tests so let's silence this phase.
-src_compile() { :; }
+# Makefile tries hard to call tests so let's silence this phase, unless we are testing.
+src_compile() {
+	if use test; then
```

Comment:
If tests only work with Python 3, please make sure to check that the Python implementation the user has chosen on his system in indeed Python 3. @mgorny wrote a function to perform this check called `python_is_python3`. You need to add it to the `if` clause:
```
if use test && python_is_python3; then
    default;
else
    :;
fi
```

Also, can we move this code to the `src_test` phase? It would be more appropriate to run tests during this phase.